rust/connlib/tunnel/src/dns.rs

```diff
@@ -333,6 +333,7 @@ impl StubResolver {
                 vec![AllRecordData::Ptr(domain::rdata::Ptr::new(fqdn))]
             }
             (Rtype::HTTPS, Some(_)) => {
+                // TODO: Should we generate a response here instead of just discarding?
```

Comment:
Why don't we just answer it with NXDOMAIN or something like that? Or empty records?